dhis-2/dhis-services/dhis-service-core/src/main/resources/org/hisp/dhis/program/hibernate/SingleEvent.hbm.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0"?>
+<!DOCTYPE hibernate-mapping PUBLIC
+  "-//Hibernate/Hibernate Mapping DTD 3.0//EN"
+  "http://www.hibernate.org/dtd/hibernate-mapping-3.0.dtd">
+
+<hibernate-mapping>
+  <class name="org.hisp.dhis.program.SingleEvent" table="event">
+
+    <id name="id" column="eventid">
+      <generator class="sequence">
+        <param name="sequence_name">programstageinstance_sequence</param>
+      </generator>
+    </id>
+
+    <property name="uid" column="uid" unique="true" length="11" />
+
+    <property name="code" column="code" not-null="false" unique="true" length="50" />
```

Comment:
are these identifiable fields not usually embedded/imported from some other hibernate file?

Reply:
I think we cannot use that file here because of the different mapping on `lastUpdatedBy` https://github.com/dhis2/dhis2-core/blob/3180693f6364583f82dc99a39c37a02c3e81ed26/dhis-2/dhis-services/dhis-service-core/src/main/resources/org/hisp/dhis/common/identifiableProperties.hbm#L5

We are using `lastUpdatedByUserInfo` in `event` table.

Reply:
A bit confusing though as Event is a BaseIdentifiableObject/IdentifiableObject so should it not have the identifiable properties 🤷🏻